ansible/roles/stack-sunbird/templates/sl-assessments.env

```diff
@@ -0,0 +1,145 @@
+# Service Config
+HOST={{sl_assessments_host | default("localhost")}}
+PORT={{sl_assessments_port | default("7201")}} 
+LOG={{sl_assessments_log | default("debug")}}
+NODE_ENV={{sl_assessments_node_env}}
+ENABLE_DEBUG_LOGGING={{sl_assessments_enable_debug_logging | default("ON")}}
+ENABLE_BUNYAN_LOGGING={{sl_assessments_enable_bunyan_logging | default("OFF")}}
+APPLICATION_BASE_URL={{sl_assessments_application_base_url | default('/assessment/')}}
+MOBILE_APPLICATION_APP_TYPE={{sl_assessments_mobile_application_app_type | default("assessment")}}
+APPLICATION_BASE_HOST={{sl_assessments_application_base_host}}
+
+# Allow report request to run for longer time
+request_timeout_for_reports={{sl_assessments_request_timeout_for_reports | default("600000")}}
+SHIKSHALOKAM_USER_PROFILE_FETCH_ENDPOINT={{sl_assessments_shikshalokam_user_profile_fetch_endpoint | default("/api/user/v1/read")}}
+AUTHORIZATION={{sl_assessments_authorization}}
+MONGODB_URL={{sl_assessments_mongodb_url}}
+SHIKSHALOKAM_BASE_HOST={{sl_assessments_shikshalokam_base_host}}
+
+#Cloud Storage Configuration
+CLOUD_STORAGE={{sl_assessments_cloud_storage}}
+
+
+# Azure Cloud Configuration
+AZURE_ACCOUNT_NAME={{sl_assessments_azure_account_name}}
+AZURE_ACCOUNT_KEY={{sl_assessments_azure_account_key}}
+AZURE_STORAGE_CONTAINER={{sl_assessments_azure_storage_container)}}
+
+
+# Google Cloud Configuration
+GCP_PATH={{sl_assessments_gcp_path}}
+GCP_BUCKET_NAME={{sl_assessments_gcp_bucket_name)}}
+
+
+# AWS Cloud Configuration
+AWS_ACCESS_KEY_ID={{sl_assessments_aws_access_key_id}}
+AWS_SECRET_ACCESS_KEY={{sl_assessments_aws_secret_access_key}}
+AWS_BUCKET_NAME={{sl_assessments_aws_bucket_name}}
+AWS_BUCKET_ENDPOINT={{sl_assessments_aws_bucket_endpoint}}
+AWS_BUCKET_REGION={{sl_assessments_aws_bucket_region}}
+
+#Current in use database name   
```

Comment:
What's this DB? is it postgres or cassandra ? How to connect to the db ?

Reply:
it is cassandra, but details for this i have to check with developer

Reply:
i have did the changes rajesh

Reply:
If you don't have the complete idea, please don't mark it as resolved.